app/services/permissions/order.rb

```diff
@@ -44,7 +44,7 @@ def editable_line_items
     def filtered_orders(orders)
       return orders unless filter_orders?
 
-      orders.complete.not_state(:canceled).ransack(search_params).result
+      orders.complete.not_state(:canceled, :shipped).ransack(search_params).result
```

Comment:
I think we probably don't want to add this here. This class and it's methods are used in lots of other places in the codebase, where we want to show shipped orders. The changes to the search filters in `line_items_controller.js` should be enough?

Reply:
You're right. The weird thing is, I added this in because it wasn't working the first time. I was probably mistaken though.